src/components/SidePane/index.tsx

```diff
@@ -0,0 +1,91 @@
+import {substituteRouteParameters} from '@libs/SidePaneUtils';
+import NAVIGATORS from '@src/NAVIGATORS';
+import getHelpContent from './getHelpContent';
+
+function SidePane({shouldShowOverlay = false}: {shouldShowOverlay?: boolean}) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const {isExtraLargeScreenWidth, shouldUseNarrowLayout} = useResponsiveLayout();
+    const {sidePaneTranslateX, shouldHideSidePane, shouldHideSidePaneBackdrop} = useSidePane();
+
+    const {route, isInNarrowPaneModal} = useNavigationState((state) => {
+        const params = (findFocusedRoute(state)?.params as Record<string, string>) ?? {};
+        const activeRoute = Navigation.getActiveRouteWithoutParams();
+        return {route: substituteRouteParameters(activeRoute, params), isInNarrowPaneModal: state.routes.some((r) => r.name === NAVIGATORS.RIGHT_MODAL_NAVIGATOR)};
+    });
+
+    const onClose = useCallback(
+        (shouldUpdateNarrow = false) => {
+            triggerSidePane(false, {shouldOnlyUpdateNarrowLayout: !isExtraLargeScreenWidth || shouldUpdateNarrow});
+        },
+        [isExtraLargeScreenWidth],
+    );
+
+    const sizeChangedFromLargeToNarrow = useRef(!isExtraLargeScreenWidth);
+    useEffect(() => {
+        // Close the side pane when the screen size changes from large to small
+        if (!isExtraLargeScreenWidth && !sizeChangedFromLargeToNarrow.current) {
+            onClose(true);
+            sizeChangedFromLargeToNarrow.current = true;
+        }
+
+        // Reset the trigger when the screen size changes back to large
+        if (isExtraLargeScreenWidth) {
+            sizeChangedFromLargeToNarrow.current = false;
+        }
+    }, [isExtraLargeScreenWidth, onClose]);
+
+    if (shouldHideSidePane) {
+        return null;
+    }
+
+    return (
+        <>
+            <View>
+                {shouldShowOverlay && !shouldHideSidePaneBackdrop && !isInNarrowPaneModal && (
+                    <Backdrop
+                        onBackdropPress={onClose}
+                        style={styles.sidePaneOverlay}
+                    />
+                )}
+            </View>
+            <Animated.View style={[styles.sidePaneContainer(shouldUseNarrowLayout, isExtraLargeScreenWidth), {transform: [{translateX: sidePaneTranslateX.current}]}]}>
+                <ScreenWrapper testID={SidePane.displayName}>
+                    <HeaderWithBackButton
+                        title={translate('common.help')}
+                        style={styles.headerBarDesktopHeight}
+                        onBackButtonPress={() => onClose(false)}
+                        onCloseButtonPress={() => onClose(false)}
+                        shouldShowBackButton={!isExtraLargeScreenWidth}
+                        shouldShowCloseButton={isExtraLargeScreenWidth}
+                        shouldDisplayHelpButton={false}
+                    />
+                    {getHelpContent(styles, route)}
```

Comment:
The help content depends on route value, while loading route will briefly become undefined which makes the help content return empty. We had an issue for it here: https://github.com/Expensify/App/issues/62215

And We fixed it here: https://github.com/Expensify/App/pull/62349